src/engine/source/api/include/api/test/handlers.hpp

```diff
@@ -19,6 +20,9 @@ constexpr auto FILTER_CONTENT_FORMAT =
 constexpr auto TEST_FILTER_FULL_NAME_FORMAT = "filter/{}_filter/0"; ///< Filter name format, '{}' is the session name
 constexpr auto TEST_POLICY_FULL_NAME_FORMAT = "policy/{}_policy/0"; ///< Policy name format, '{}' is the session name
 constexpr auto TEST_ROUTE_NAME_FORMAT = "{}_route";                 ///< Route name format, '{}' is the session name
+constexpr auto EVENT_CONTENT_FORMAT =
+    R"({{"wazuh":{{"queue": {}, "location": "{}", "message": "{}"}}, "~TestSessionName": "{}"}})";   ///< Event content format
+constexpr auto DEFAULT_PROTOCOL_LOCATION {"/dev/stdin"}; ///< Default protocol location
```

Comment:
Wouldn't it be better to represent the location with something related to the API? I would also define it with the "TEST_" prefix. The same goes for `EVENT_CONTENT_FORMAT`